libcloudforensics/gcp.py

```diff
@@ -494,19 +493,19 @@ def __list_by_label(self, labels_filter, service_object, filter_union):
     """
     if not isinstance(filter_union, bool):
       raise RuntimeError((
-          'filter_union parameter must be of Type boolean'
-          ' {0:s} is an invalid argument.').format(filter_union))
+                           'filter_union parameter must be of Type boolean'
+                           ' {0:s} is an invalid argument.').format(filter_union))
```

Comment:
There seems to be an extra set of `()` around this string.

Reply:
Addressed